content/news/buzz/project-spotlight-colosseum/contents+es.lr

```diff
@@ -0,0 +1,33 @@
+﻿title: Proyecto Spotlight: Colosseum
```

Comment:
Lets make the title spanish as well

`Proyecto destacado: Colosseum`

Also on the admin interface make sure to update the slug so the URL of the page is also on Spanish.